src/bindings/python/src/pyopenvino/test_utils/test_utils.cpp

```diff
@@ -13,20 +13,52 @@ namespace py = pybind11;
 PYBIND11_MODULE(test_utils_api, m) {
     m.def(
         "compare_functions",
-        [](const ov::Model& lhs, const ov::Model& rhs, bool compare_tensor_names) {
+        [](const ov::Model& lhs,
+           const ov::Model& rhs,
+           bool compare_const_values,
+           bool compare_names,
+           bool compare_runtime_keys,
+           bool compare_precisions,
+           bool compare_attributes,
+           bool compare_tensor_names,
+           bool compare_accuracy,
+           bool compare_subgraph_descriptors,
+           bool compare_consumers_count) {
             const auto lhs_ptr = std::const_pointer_cast<ov::Model>(lhs.shared_from_this());
             const auto rhs_ptr = std::const_pointer_cast<ov::Model>(rhs.shared_from_this());
 
-            auto fc = FunctionsComparator::with_default()
-                          .enable(FunctionsComparator::ATTRIBUTES)
-                          .enable(FunctionsComparator::CONST_VALUES);
+            auto fc = FunctionsComparator::no_default().enable(FunctionsComparator::NODES);
```

Comment:
I believe it would be better to support enums in Python. For the reference, please, look [here](https://pybind11.readthedocs.io/en/stable/classes.html#enumerations-and-internal-types) and [here](https://github.com/openvinotoolkit/openvino/blob/master/src/bindings/python/src/pyopenvino/graph/passes/transformations.cpp#L26).